apps/cms/news/templates/a4_candy_cms_news/news_index_page.html

```diff
@@ -15,7 +15,7 @@ <h1 class="cms__title--serif cms__title--underlined text-center">{{ page.subtitl
                 {% for post in news %}
                     <div class="col-12 col-md-6 cms-news__tile">
                         <a class="cms__tile-link" href="{% pageurl post %}">
-                            <div class="cms__text text-muted">
+                            <div class="cms__text">
                                 {{ post.first_published_at|date }}
```

Comment:
in design date is grey, has it changed?

Reply:
The categories afor use-cases are grey, but the dates for news are not.